src/helpers/useMultiStepForm.tsx

```diff
@@ -10,13 +10,20 @@ function useMultiStepForm(steps: Pages, lastPage: JSX.Element) {
   const [curStep, setCurStep] = useState(0)
   const pages = Object.keys(steps)
 
+  const scrollToTop = () => {
+    setTimeout(() => {
+      window.scrollTo({ top: 0, behavior: 'smooth' })
```

Comment:
Kanskje bare meg, men synes det er litt forstyrrende at siden først endrer seg, og så blir man scrollet til toppen. Kan hende det gir en bedre opplevelse å ikke bruke `behavior: 'smooth'`, for da skjer både endring av innhold og scrolling så godt som samtidig. Dere kan vurdere det selv.

Reply:
Jeg er enig her :D Endrer